### DISCUSSION TITLE
Question: 閲覧権限の無いページを lsx や最新の変更リストに表示しない方法

### DISCUSSION THREAD

**Discussion by aviscaerulea** [growilabs/growi#7290] [Help / Questions] [answered]
「自分だけ」参照可能な制限ページを作成しても lsx やサイドバーの「最新の変更」にページタイトルが表示されてしまいます。

そうとは気付かないユーザがタイトルに保護したい文言などを記載していると問題になるので、ページタイトルもわざわざダミーのものにしなければなりません。

また読めないページが lsx や「最近の更新」に表示されても権限のないユーザにはノイズになってしまうので、ここら辺を描画前にフィルタして欲しいです。

| item     | version |
| ---      | --- |
|OS        |Amazon Linux2|
|GROWI     |5.0.4|
|node.js   |16.15.0|
|npm       |8.5.5|
|yarn      |1.22.18|
|Using Docker|yes|
|Using [growi-docker-compose][growi-docker-compose]|yes|

**Comment by yuki-takei** [comment, marked as answer]
@aviscaerulea `/admin/security` の「ページリスト・検索結果」の設定を試してみてください。一括の設定にはなりますが表示・非表示を変更できます。

![image](https://user-images.githubusercontent.com/1638767/214238253-1d74503c-3d05-4c6a-a54c-98c3f28024c4.png)

**Comment by aviscaerulea** [comment]
@yuki-takei 
あ、すみません、これは見落としていました。
教えていただいた設定を非表示に変更することで、閲覧権限の無いページを出さないようにできました。
`$lsx` による表示からもフィルタされることを確認しました。

ちなみに、意図は察せられますが「表示/非表示」とトグルボタンが位置的に逆なのが人によっては混乱するかもしれませんね。（トグルを左にすると「表示」と思うかもしれない）

**Comment by yuki-takei** [comment]
バージョン忘れましたが最近トグルの左右反転させました。クローズします。
